pip_review/__main__.py

```diff
@@ -269,57 +163,60 @@ def confirm(question):
         answer = answer.strip().lower()
     return answer == 'y'
 
+def parse_legacy(pip_output):
+    packages = []
+    for line in pip_output.splitlines():
+        package = {}
+        line = line.split(" - ")
+        package['name'] = re.findall(r'^[a-zA-Z0-9\-]+', line[0])[0]
+        package['version'] = re.findall(r'\(([0-9a-zA-Z\.]+)\)', line[0])[0]
+        package['latest_version'] = re.findall(r'(^[0-9a-zA-Z\.]+)', line[1].split(":")[1].strip())[0]
+        packages.append(package)
+    return packages
+
+
+def get_outdated_packages(unknown):
+    command = ['pip', 'list', '--outdated'] + unknown
+    if parse_version(pip.__version__) > parse_version('9.0'):
+        command.append('--disable-pip-version-check')
+        command.append('--format=json')
+        output = check_output(" ".join(command)).decode('utf-8')
```

Comment:
I agree it should be a list, but I vaguely recall that `check_output` requires a string argument. It hasn't caused troubles *yet*, so it's not stopping me from rolling out the 1.0 release, but go ahead and submit a pull request if you can't stand it.

Your fix should branch off from `feature/pip-list` but you should request to merge into `develop`, because I locally already merged the branch. Thanks in advance if you do, no problem if you don't. I'm fine with it either way.